packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss

```diff
@@ -29,13 +29,29 @@
         padding-right: 1rem;
     }
     &-card-container {
+        position: relative;
+        overflow: hidden;
         width: 27rem;
         border: 1px solid var(--general-hover);
-        padding: 2.4rem;
         border-radius: 2.4rem;
         &:hover {
             box-shadow: 0 2px 8px 0 var(--shadow-menu);
         }
+        &__banner {
+            position: absolute;
+            z-index: 50;
```

Comment:
I suggest using 9999 to have it on top everything which might be added later. either the highest number or a 1 digit number. number 50 is so randomly chosen